### PR TITLE
Compute ALOS

### PR DESCRIPTION
**Pull request by edknock**

### CODE REVIEW

**Review by RaphaelS1 — COMMENTED** [review]
LGTM but looks like tests only check the simple case of no vacc and 1 strain; is it possible to add tests for other cases?

**Review by richfitz — APPROVED** [review]
This is fine for code, though I can't comment on the ALOS calculation irtself - @MJomaba do you have more to add

File: DESCRIPTION [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Package: sircovid
 Title: SIR Model for COVID-19
-Version: 0.11.9
+Version: 0.11.10
```
Comment (richfitz) [inline DESCRIPTION]:
```suggestion
Version: 0.11.11
```

**Review by richfitz — CHANGES_REQUESTED** [review]
Ah, after reviewing I noticed the version had moved past us. I've updated the news, and here's a change for the version number

**Review by richfitz — APPROVED** [review]

**Pull request merged**